website/www/site/content/en/blog/adding-data-sources-to-sql.md

```diff
@@ -81,19 +81,19 @@ The `TableProvider` classes are under
 
 Our table provider looks like this:
 
-{{< highlight java >}}
+{{< highlight language="java" >}}
 @AutoService(TableProvider.class)
 public class GenerateSequenceTableProvider extends InMemoryMetaTableProvider {
 
-  @Override
-  public String getTableType() {
-    return "sequence";
-  }
+@Override
+public String getTableType() {
+return "sequence";
+}
 
-  @Override
-  public BeamSqlTable buildBeamSqlTable(Table table) {
-    return new GenerateSequenceTable(table);
-  }
+@Override
+public BeamSqlTable buildBeamSqlTable(Table table) {
+return new GenerateSequenceTable(table);
+}
```

Comment:
I don't think we should be making changes to a blog post's contents. It looks like you've made changes to a lot of blog posts to remove leading whitespace in code snippets.

Are these changes something you did while you were experimenting? Can they be reverted?

Reply:
Yeah, I spotted this later on and turned off the formatter.
About ``` I changed it to `<< highlight >>`, because new code snippet has copy option and wanted to add this to every code

Reply:
the content in each file in documentation wasn't changed, only this code snippets were changed

Reply:
Can we just set up the website to use the new code snippet widget with ```? Unless we add some explicit check to forbid ```, people will continue to use that syntax since it's very common markdown syntax.